### ISSUE TITLE
AVT 1 - Pagination & Multiple Pagination have DAP violation for missing labels

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4534]
## Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v10 - React
DAP IBM Accessibility 1.8.0.0 Sept 2019 Ruleset

## Detailed Description
Go to the React Component 
Run DAP and the following violations are present (see screenshot):
<img width="947" alt="Screen Shot 2019-11-01 at 3 33 07 PM" src="https://user-images.githubusercontent.com/21676914/68054877-5c5b0c00-fcbd-11e9-8730-5cef6833c107.png">

**Comment by joshblack** [comment]
This seems to be resolved in the latest storybook, going to close this out unless there are some additional steps to reproduce this bug that I did not perform! Only violation I found is related to storybook env:

![image](https://user-images.githubusercontent.com/3901764/69454961-019d5900-0d2d-11ea-9ce5-e58d50b22486.png)

**Comment by snidersd** [comment]
@joshblack I retested today (see screenshot) and still see the violation. Can you check the DAP ruleset your testing against? 
<img width="1348" alt="Screen Shot 2019-11-22 at 4 31 58 PM" src="https://user-images.githubusercontent.com/21676914/69465188-0faba380-0d46-11ea-89fd-86aad32fead7.png">

**Comment by joshblack** [comment]
@snidersd for sure! Tested with the following ruleset:

![image](https://user-images.githubusercontent.com/3901764/69518859-d1fd7500-0f15-11ea-8ee7-1bcda9ffa7a3.png)

At the URL: https://carbon-components-react.netlify.com/?path=/story/pagination--pagination

With the following results:

![image](https://user-images.githubusercontent.com/3901764/69518991-2d2f6780-0f16-11ea-9fa0-8dba0610db12.png)
